SECURITY.md

```diff
@@ -6,55 +6,63 @@ If you believe you have found a security vulnerability in any Magma Core Foundat
 
-  * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
-  * Step-by-step instructions to reproduce the issue
-  * Any special configuration required to reproduce the issue
-  * Impact of the issue, including how an attacker might exploit the issue
-  * Proof-of-concept or exploit code (if possible)
-  * Full paths of source file(s) related to the manifestation of the issue
-  * The location of the affected source code (tag/branch/commit or direct URL).
+- Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
+- Step-by-step instructions to reproduce the issue
+- Any special configuration required to reproduce the issue
+- Impact of the issue, including how an attacker might exploit the issue
+- Proof-of-concept or exploit code (if possible)
+- Full paths of source file(s) related to the manifestation of the issue
+- The location of the affected source code (tag/branch/commit or direct URL).
+
+We prefer all communications to be in English.
 
 ## Response Process
 
 Certain members of the Magma team have been designated as the "vulnerability management team" and will receive any e-mails sent to `security@magmacore.org`. When receiving such an e-mail, they will:
 
-1. Reply to the e-mail acknowledging its receipt, cc'ing `security@magmacore.org` so that the other members of the team are aware that they are handling the issue.  You should receive this initial response within 72 hours, usually sooner. If for some reason you do not, please follow up via email to ensure we received your original message.
+1. Reply to the e-mail acknowledging its receipt, cc'ing `security@magmacore.org` so that the other members of the team are aware that they are handling the issue. If you are a submitter and do not receive a response within 72 hours, please follow up via email to ensure we received your original message.
 
-2. Security issues will be evaluated by the Security Vulnerability team and assigned severity level using the Common Vulnerability Scoring System as defined by [First.org](https://first.org/cvss/user-guide).  Development of remediation and fixes shall be prioritized as follows:
+2. Inform the #security channel in [the Magma Slack](https://magmacore.slack.com)
 
-   - Critical Risk: shall be treated as "P0" issue, with a fix developed and released as soon as possible even if it requires an out-or-sequence release.
-   - High Risk: shall be treated as "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
-   - Medium Risk: shall be treated a "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
-   - Low Risk: shall be treated as a "P2" issue, with a fix delivered no later than the next scheduled "major" release
+2. Evaluate severity using the [Scoring Rubric Template](https://docs.google.com/spreadsheets/d/1iOOgdmfHImngHE9SsOH6TV_BfzZ34C4MSKE6kaY_nRI/edit#gid=0).
```

Comment:
Within the file it states:
`WARNING: THIS DOCUMENT IS AN OLD VERSION. IT HAS BEEN REPLACED BY ...`

I think we should not link to an old outdated version of any document.

Reply:
Thanks for the catch. Fixed.

Reply:
I have reviewed fixed indentation and suboptimal newlines.